packages/atomic/src/pages/index.html

```diff
@@ -211,6 +211,9 @@
             </atomic-sort-dropdown>
             <atomic-did-you-mean></atomic-did-you-mean>
           </atomic-layout-section>
+          <atomic-layout-section section="notification"
```

Comment:
I'm not entirely convinced a new section is needed especially since there's only this component inside (requires a doc update + our customer to implement it), it could easily fit in either results or status.

Reply:
If it doesn't have its own section, then I'll add it in `status`.

Reply:
Indeed, adding sections has a "cost" (doc, maintenance, burden on implementers). 

It needs to be really worth it before we add a new one.